lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js

```diff
@@ -0,0 +1,56 @@
+import moment from 'moment'
+import { formatDuration } from '@opentripplanner/core-utils/lib/time'
+
+export default function baseRenderer (monitoredTrip) {
```

Comment:
This is a good concept. The term `renderer` doesn't quite fit because the function doesn't actually return any JSX, but I can't think of an alternate right now...